client/my-sites/plans-features-main/test/plan-type-selector.jsx

```diff
@@ -17,25 +24,36 @@ describe( '<PlanTypeSelector />', () => {
 	};
 
 	test( 'Should show CustomerTypeToggle when kind is set to `customer`', () => {
-		const comp = shallow(
-			<PlanTypeSelector { ...myProps } kind="customer" customerType="personal" />
-		);
+		render( <PlanTypeSelector { ...myProps } kind="customer" customerType="personal" /> );
+
+		expect( screen.getByRole( 'radiogroup' ) ).toHaveClass( 'is-customer-type-toggle' );
+
+		const radios = screen.queryAllByRole( 'radio' );
 
-		expect( comp.find( 'CustomerTypeToggle' ).length ).toBe( 1 );
+		expect( radios[ 0 ] ).toHaveTextContent( 'Blogs and personal sites' );
+		expect( radios[ 1 ] ).toHaveTextContent( 'Business sites and online stores' );
 
-		// customerType prop is passed to the child component
-		expect( comp.find( 'CustomerTypeToggle[customerType="personal"]' ).length ).toBe( 1 );
```

Comment:
Should we port this test too? I think it can be useful. Should be easy to achieve if we mock `CustomerTypeToggle` and render the `customerType` prop somewhere in it.

Reply:
It is, though not literally. I tested for whether the appropriate radio is checked which is a direct result of passing on `customerType = 'personal'`. Second usage is in `generatePath` which is tested for in another test below.